src/components/SelectionList/BaseListItem.tsx

```diff
@@ -42,14 +40,14 @@ function BaseListItem({
         >
             <PressableWithFeedback
                 onPress={() => onSelectRow(item)}
-                disabled={isDisabled}
+                disabled={item.isDisabled}
                 accessibilityLabel={item.text}
                 role={CONST.ROLE.BUTTON}
                 hoverDimmingValue={1}
                 hoverStyle={styles.hoveredComponentBG}
                 dataSet={{[CONST.SELECTION_SCRAPER_HIDDEN_ELEMENT]: true}}
                 onMouseDown={shouldPreventDefaultFocusOnSelectRow ? (e) => e.preventDefault() : undefined}
-                nativeID={keyForList}
+                testID={keyForList}
```

Comment:
Why was this changed?

Reply:
It was done by Kuba Szymczak so I have no idea tbh, should I revert this change?

Reply:
Yes 😄

Reply:
Reverted